### PR TITLE
[VEGA-6565] MultiFilters added to vega-table

### PR DESCRIPTION
**Pull request by Alexandr1111**
## Задача
Cсылка в Jira GPN на задачу: https://jira.konakov.tv/browse/VEGA-6565

## Problem statement
Добавлена возможность фильтрации строк таблицы по нескольким фильтрам в совокупности(фасеточные фильтры)

https://artcpt.atlassian.net/browse/VEGA-6564
https://artcpt.atlassian.net/browse/VEGA-6565
https://artcpt.atlassian.net/browse/VEGA-6566
https://artcpt.atlassian.net/browse/VEGA-6567

## Solution details

_{What has been done (list of details)}_

## Type
- [x] Feature
- [ ] Bug Fix
- [x] Refactoring
- [ ] Test
- [ ] Other

## Associated issues

_{Jira issues or PRs related to this one}_

## Quality control
- [ ] PR: Based on a correct branch
- [ ] PR: Head branch is rebased on the base branch
- [ ] PR: Assignee chosen and necessary labels are set
- [ ] PR: Current form is filled out (where it makes sense)
- [ ] PR: Diff checked for irrelevant changes
- [ ] PR: Commit messages follow [the rules](../docs/commits-style.md)
- [ ] JS: There's no errors/warnings in the browser dev console
- [ ] Layout: Uses [variables](https://github.com/gpn-prototypes/ui-kit/tree/master/src/components/Theme) and [CSS classes](https://github.com/gpn-prototypes/ui-kit/blob/master/src/utils/whitepaper/whitepaper.css) from [GPN design system](https://github.com/gpn-prototypes/ui-kit)
- [ ] Layout: Tested with various content amounts
- [ ] Docs: All changes to the component API are reflected and key points are described
- [ ] Storybook: Stories are written/updated for components
- [ ] Storybook: Stories follow [the rules](https://github.com/gpn-prototypes/vega-ui/blob/master/docs/storybook.md)
- [ ] Tests: New features and bugfixes are covered with tests
- [ ] Tests: Existing tests passed successfully

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

File: src/components/vega-table/filter-popover/TableFilterPopover.tsx [hunk 1]
```diff
@@ -82,28 +79,53 @@ export const TableFilterPopover: React.FC<Props> = <
       filterValue: getFilterToSet(filterValue),
     });
 
+    console.log(filterValue, getFilterToSet(filterValue), 'grgec');
```
Comment (SergeyKatugin) [inline src/components/vega-table/filter-popover/TableFilterPopover.tsx:82]: Лог

File: src/components/vega-table/filter-popover/TableFilterPopover.tsx [hunk 2]
```diff
@@ -82,28 +79,53 @@ export const TableFilterPopover: React.FC<Props> = <
...
+          (currentFilter.filterValue as string[]).includes(option.displayName),
+        ),
+    );
+
+    // console.log(
+    //   mappedResult,
+    //   columnTextOptions,
+    //   currentFiltersValue,
+    //   'mappedResult',
+    // );
```
Comment (SergeyKatugin) [inline src/components/vega-table/filter-popover/TableFilterPopover.tsx:125]: Лишнее

File: src/components/vega-table/text-filter/TextFilter.tsx [hunk 3]
```diff
@@ -80,6 +80,8 @@ export const TableTextFilter: React.FC<Props> = ({
     [checkboxGroupValue],
   );
 
+  console.log(checkboxGroupValue, filterValue, 'checkboxGroupValue');
```
Comment (SergeyKatugin) [inline src/components/vega-table/text-filter/TextFilter.tsx:83]: Лог

File: src/components/vega-table/filter-popover/TableFilterPopover.tsx (hunk 1 above)
Comment (Alexandr1111) [reply src/components/vega-table/filter-popover/TableFilterPopover.tsx:82]: Драфт..

File: reports/junit.xml [hunk 4]
```diff
@@ -1,545 +1,545 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuites name="Jest Tests" tests="234" failures="0" errors="0" time="13.148">
-  <testsuite name="SplitPanes" errors="0" failures="0" skipped="0" timestamp="2022-09-28T05:55:09" time="4.396" tests="11">
-    <testcase classname="SplitPanes рендерится без ошибок" name="SplitPanes рендерится без ошибок" time="0.069">
+<testsuites name="Jest Tests" tests="234" failures="0" errors="0" time="13.521">
```
Comment (SergeyKatugin) [inline reports/junit.xml:2]: лишнее?

**Review by SergeyKatugin — APPROVED** [review]

**Pull request merged**